### ISSUE TITLE
Popup dimensions not working

### ISSUE THREAD

**Issue by Pauan** [Pauan/tab-organizer#151]
Originally reported on Google Code with ID 152

```
System: Mac OS X
Browser: Chrome 16.0.912
Version: Latest available in the Chrome Extension Store.

What steps will reproduce the problem?
1. Open the extensions popup;
2. Click on the "options" link;
3. Select the "Popup" tab;
4. Try to change any input value (left/right space, width, height).

What is the expected output? What do you see instead?
Inserting an integer, the popup dimensions should change. Instead, inserting any number
in the input box instead insert an empty character until the input is completely emptied.
At this point the window size became the minimal admissible, 200px x 200px. There is
no way change this value except resetting the extension configuration. -.-
```

Reported by `dnl.rlnd` on 2011-11-16 19:01:08

**Comment by Pauan** [comment]
```
Egads, you're right! I'll get that fixed right away. Thanks for the report!
```

Reported by `pcxunlimited` on 2011-11-16 21:35:54
- **Status changed**: `Delegated`

**Comment by Pauan** [comment]
```
This issue was updated by revision 8833c4e89846.
```

Reported by `pcxunlimited` on 2011-11-16 21:57:44
- **Status changed**: `Pending`

**Comment by Pauan** [comment]
```
It seems that Chrome changed the behavior of the maxLength property... in any case,
it should be fixed for now.

Please upgrade to version 4.2 (it should happen automatically) and then tell me if
the problem is fixed or not.
```

Reported by `pcxunlimited` on 2011-11-16 22:27:19
- **Status changed**: `Verified`

**Comment by Pauan** [comment]
```
Great and fast fix. I can confirm that in the 4.2 release the issue is resolved.

Just a note, I noticed that selecting the "Bubble" display mode, the width and the
height does not affect the bubble window size.
The popup mode works as expected, the tab mode obviously is not affected by the size
properties, what about the bubble mode? Is it an intended behavior or have I to report
it as a bug? It is not clear, and may be good to change the bubble size.

p.s.: I take the opportunity to congratulate with for this great extension. A very
good idea and work!
```

Reported by `dnl.rlnd` on 2011-11-17 15:12:02

**Comment by Pauan** [comment]
```
It's a known issue, but there's nothing I can do to fix it: Chrome does not allow extensions
to change the bubble size. Instead, the bubble is automatically resized (by Chrome)
to fit the contents of the bubble.

As a workaround, I have it set up so that the content's size will be as big as you
specify in the options page. To understand what I'm talking about, change the bubble's
size to 200x200. You'll note that the bubble is now very small.

Now change the bubble's size to something very big. The bubble is now bigger than it
was at 200x200, but is still smaller than expected. That's because Chrome puts a maximum
limit on the size of the bubble, and there's nothing I can do about it.

tl;dr: Chrome won't let me change the bubble's size
```

Reported by `pcxunlimited` on 2011-11-17 16:26:33

**Comment by Pauan** [comment]
```
I filed issue 152 for that. Please star that issue if you want to be notified when it's
fixed.
```

Reported by `pcxunlimited` on 2011-11-17 16:41:06

**Comment by Pauan** [comment]
Reported by `pcxunlimited` on 2012-08-27 23:44:48
- **Status changed**: `Fixed`
